addons/mail/static/src/new/thread/message_in_reply_to.scss

```diff
@@ -0,0 +1,35 @@
+.o-mail-message-in-reply-corner {
+    &.o-isLeftAlign {
+        left: $o-mail-message-sidebar-width / 2;
+        border-radius: $o-mail-rounded-rectangle-border-radius-sm 0 0 0;
+    }
+
+    &.o-isRightAlign {
+        right: $o-mail-message-sidebar-width / 2;
+        border-radius: 0 $o-mail-rounded-rectangle-border-radius-sm 0 0;
+    }
+}
+
+.o-mail-message-in-reply-avatar {
+    width: $o-mail-thread-avatar-size / 2;
+    height: $o-mail-thread-avatar-size / 2;
+}
+
+.o-mail-message-in-reply-wrap-inner {
+    display: -webkit-box;
+    -webkit-box-orient: vertical;
+    -webkit-line-clamp: 2;
+}
+
+.o-mail-message-in-reply-body {
+    // Make the body single line when possible
+    p, div {
+        display: inline;
+        margin: 0;
+    }
+
+    br {
+        display: none;
+    }
```

Comment:
Why not use `@mail/new/utils/htmlToTextContentInline`?

Reply:
It wouldn't behave as it does in master. For example, linked are not preserved while they should be

Reply:
Ok